### DISCUSSION TITLE
Welcome to dupeguru Discussions!

### DISCUSSION THREAD

**Discussion by arsenetar** [hsoft/dupeguru#931] [Announcements]
## 👋 Welcome!
Adding discussions as a way to better address items that do not fit into Issues workflow.  This includes general ideas that may not be a feature request as well as Q&A.

**Comment by PatchNoir** [comment]
I created an github account just to post here. I am a brasilian end user of the dupe guru and it is the one of the best duplicate removers  i ever used, and i will continue using it for a loong time. Thanks for everyone involved on this project since the beggining, its rare to see an dupe finder with an "content comparison" mode. The only thing i would want (if its not a problem) its to be able to change the % on the filter sensibility, because when comparing by content he is stuck at 95%. The program use byte-byte comparison, hash, or another thing like sha ?

**Reply by arsenetar** [reply]
Filter hardness is used for the filename comparisons only right now (which is why it is grayed out for content scan).  The files have md5 hashes calculated and compared for content scans (or in the upcoming version partial hashes for large files).
